documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-verify.adoc
+// getting-started.adoc
+
+[id='setup-external-clients-{context}']
+= Setting up access for clients outside of Kubernetes
+
+This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes.
+
+Using the address of the Kafka cluster, you can provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
+
+You configure an external Kafka listener to provide the access.
+
+The following external listener types are supported:
+
+* `route` to use OpenShift `Route` and the default HAProxy router
+* `loadbalancer` to use loadbalancer services
+* `nodeport` to use ports on Kubernetes nodes
+* `ingress` to use Kubernetes _Ingress_ and the {NginxIngressController}
+
+The type chosen depends on your requirements, and your environment and infrastructure.
+For example, `nodeport` is the least secure, so you might not wish to use it in a production environment.
+
+In this procedure:
+
+. An external listener is configured for the Kafka cluster, with TLS encryption and authentication, and Kafka _simple authorization_ is enabled.
+. A `KafkaUser` is created for the client, with TLS authentication and Access Control Lists (ACLs) defined for _simple authorization_.
+
+You can configure your listener to use TLS or SCRAM-SHA authentication,
+both of which can be used with TLS encryption.
+If you are using an authorization server, you can use token-based link:{BookURLUsing}#assembly-oauth-authentication_str[{oauth} authentication] and link:{BookURLUsing}#assembly-oauth-authorization_str[{oauth} authorization].
+
+When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent Kafka configuration:
+
+* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
```

Comment:
I don't know if it's clear for users. It should say that you should have at least one listener supporting the authentication you want to use for the KafkaUser.

Reply:
Thanks. I've added as sentence after the bullets, 

_You should have at least one listener supporting the authentication you want to use for the `KafkaUser`._